runtime/vm/createramclass.cpp

```diff
@@ -1719,6 +1719,15 @@ loadSuperClassAndInterfaces(J9VMThread *vmThread, J9ClassLoader *classLoader, J9
 
 				for (i = 0; i<romClass->interfaceCount; i++) {
 					J9UTF8 *interfaceName = NNSRP_GET(interfaceNames[i], J9UTF8*);
+					
+					if (0 == compareUTF8Length(J9UTF8_DATA(interfaceName), J9UTF8_LENGTH(interfaceName),
```

Comment:
Is this a general case we've been missing?  That applies across all releases?

Reply:
Yes, I think it is a general case we are missing. A class implements an interface the same as its own name.